### PR TITLE
Add placeholder parameter to updateTextInput

### PR DESCRIPTION
**Pull request by carlganz**
Related to #1738 

Let me know if want any changes.

### CODE REVIEW

File: R/update-input.R [hunk 1]
```diff
@@ -34,15 +35,16 @@
...
+updateTextInput <- function(session, inputId, label = NULL, value = NULL, placeholder = NULL) {
+  message <- dropNulls(list(label=label, value=value, placeholder=placeholder))
   session$sendInputMessage(inputId, message)
 }
 
 #' Change the value of a textarea input on the client
 #'
 #' @template update-input
 #' @param value The value to set for the input object.
+#' @param placeholder The placeholder to set for the input object.
```
Comment (wch) [inline R/update-input.R]: You can actually remove both of these params, and use `@inheritParams updateTextInput`.

**Comment by wch** [comment]
@carlganz Have you filled out a contributor agreement? I looked at the list that we have and don't see you on it. See https://github.com/rstudio/shiny/blob/master/CONTRIBUTING.md

**Comment by carlganz** [comment]
Fixed the roxygen, and I sent JJ a copy of the contributor agreement.

**Comment by wch** [comment]
One more thing: can you also add an entry to NEWS.md? It can go in the "Minor new features and improvements" section, and have the format, "Addressed #1738: text here (#1742)"

**Comment by carlganz** [comment]
Added a news post. Let me if you need any other changes.

**Comment by wch** [comment]
Looks like there's a merge conflict now, because of the files generated by Grunt. Can you rebase your branch and delete the grunt commit? We'll just run grunt ourselves after it's merged into master.

**Pull request merged**